beacon-chain/node/node.go

```diff
@@ -397,17 +397,19 @@ func initSyncWaiter(ctx context.Context, complete chan struct{}) func() error {
 	}
 }
 
-func newRouter(cliCtx *cli.Context) *mux.Router {
+func newRouter(cliCtx *cli.Context)http.Handler {
 	var allowedOrigins []string
 	if cliCtx.IsSet(flags.HTTPServerCorsDomain.Name) {
 		allowedOrigins = strings.Split(cliCtx.String(flags.HTTPServerCorsDomain.Name), ",")
 	} else {
 		allowedOrigins = strings.Split(flags.HTTPServerCorsDomain.Value, ",")
 	}
-	r := mux.NewRouter()
-	r.Use(middleware.NormalizeQueryValuesHandler)
-	r.Use(middleware.CorsHandler(allowedOrigins))
-	return r
+	r := http.NewServeMux()
+	// r.Use(middleware.NormalizeQueryValuesHandler)
+	handler := middleware.NormalizeQueryValuesHandler(
```

Comment:
we probably want some kind of idiomatic way of handling this for future middleware looking through a list of these. also there is a comment above